numpy/distutils/misc_util.py

```diff
@@ -2285,9 +2285,11 @@ def generate_config_py(target):
     # For gfortran+msvc combination, extra shared libraries may exist
     f.write("""
 import os
+import sys
+
 extra_dll_dir = os.path.join(os.path.dirname(__file__), '.libs')
-if os.path.isdir(extra_dll_dir):
-    os.environ["PATH"] += os.pathsep + extra_dll_dir
+if sys.platform == 'win32' and os.path.isdir(extra_dll_dir):
+    os.environ["PATH"] = os.pathsep.join(os.environ.get("PATH", "").split(os.pathsep) + [extra_dll_dir])
```

Comment:
100 char > 79 char. Why do you split the path?

Reply:
Note that with the refault path `""`, you still end up with a `os.pathsep` prefixing the `extra_dll_dir`. Is that OK?

Reply:
Suggested by @eric-wieser. I think I like the previous version better.

Reply:
> Why do you split the path?

An attempt to not end up with that extra pathsep. I thought that `"".split(":")` would return the empty list, but it does not.